examples/quickstart-concurrent/.gitignore

```diff
@@ -0,0 +1,6 @@
+Dockerfile
+.devspace/
+chart/
+node_modules/
+test/
+devspace.yaml
```

Comment:
Please remove devspace.yaml from .gitignore. It should be part of the example.

Reply:
My bad, I'll clean that up. I just duplicated it from quickstart and didn't notice the gitignore included devspace.yaml.

Reply:
Thank you for resolving it.